### DISCUSSION TITLE
Resizing Options

### DISCUSSION THREAD

**Discussion by Slackow** [PintaProject/Pinta#596] [Ideas]
I've been using Paint.net for a while for pixel art, and this program does a very good job at replicating many of it's functions, but one feature I am missing quite a lot is the ability to resize images with "nearest neighbor" instead of the bilinear algorithm. This is much more useful for pixel art. Not sure if this has been suggested before or tried, but it would be a much appreciated addition!

**Comment by cameronwhite** [comment]
This probably is pretty easy to add, since Cairo already has a nearest neighbor sampling option that we use when zooming in the canvas: https://github.com/PintaProject/Pinta/blob/4c209cd397ebe0626ebb0c76651dcfc25dd827ad/Pinta.Gui.Widgets/Widgets/Canvas/CanvasRenderer.cs#L91

**Comment by cameronwhite** [comment]
Added in 6e5545db

**Reply by Slackow** [reply]
Thank you! appreciate your time working on this.
